unittests/eosio.token_tests.cpp

```diff
@@ -299,7 +299,7 @@ BOOST_FIXTURE_TEST_CASE( transfer_tests, eosio_token_tester ) try {
       transfer( N(alice), N(bob), asset::from_string("-1000 CERO"), "hola" )
    );
 
-   BOOST_REQUIRE_EQUAL( error( "condition: assertion failed: attempt to subtract asset with different symbol" ),
+   BOOST_REQUIRE_EQUAL( error( "condition: assertion failed: asset symbol has higher precision than expected" ),
```

Comment:
Why are we expecting the error message for too high of precision?  It sure seems like a mismatch with the data that is being passed.

Reply:
With PR #2735, transfer should accept different precisions as long as the symbol name is the same and the precision of the input is no greater than the precision defined for the asset. You can see in currency_tests/test_input_quantity how lower precision assets with the same symbol name work just fine with transfer now.